### PR TITLE
remove docs saying links are to entries

### PR DESCRIPTION
**Pull request by thedavidmeister**
### TODO:
- [ ] CHANGELOG(s) updated with appropriate info
- [ ] Just before pressing the merge button, ensure new entries to CHANGELOG(s) are still under the _UNRELEASED_ heading

### CODE REVIEW

File: crates/hdk/src/link.rs [hunk 1]
```diff
@@ -33,16 +32,15 @@ pub use hdi::link::*;
 /// See [ `get_details` ] and get for more information about CRUD
 /// See [ `get_links` ] and [ `get_link_details` ] for more information about filtering by tag
 ///
-/// Generally links and CRUDs _do not interact_ beyond the fact that links need entry hashes to
+/// Generally links and CRUDs _do not interact_ beyond the fact that links need hashes to
```
Comment (jost-s) [inline crates/hdk/src/link.rs:35]: I don't understand what "interact" means. CRUDs being create and update actions, say I have a link to an entry. That entry has been created successfully. The entry is the referenced data.
What does interact mean here?

Comment (thedavidmeister) [reply crates/hdk/src/link.rs:35]: @jost-s they don't interact, i don't have a specific definition of something they don't do

Comment (jost-s) [reply crates/hdk/src/link.rs:35]: Does it mean that the link to a hash is not affected by any CRUD operation that could be applied to the link target?

Comment (thedavidmeister) [reply crates/hdk/src/link.rs:35]: @jost-s yeah that's part of not interacting, links simply don't care about CRUD and CRUD doesn't care about links

Comment (jost-s) [reply crates/hdk/src/link.rs:35]: Reading this paragraph again makes it clear for me.

File: crates/hdk/src/link.rs [hunk 2]
```diff
@@ -33,16 +32,15 @@ pub use hdi::link::*;
...
 /// reference for the base and target to already exist due to a prior create or update.
-/// The entry value only needs to exist on the DHT for the link to validate, it doesn't need to be
+/// The referenced data only needs to exist on the DHT for the link to validate, it doesn't need to be
 /// live and can have any combination of valid/invalid crud actions.
```
Comment (jost-s) [inline crates/hdk/src/link.rs:38]: A link needs to exist on the DHT but doesn't need to be live. What does "live" mean?

**Review by jost-s — APPROVED** [review]

File: crates/hdk/src/link.rs (hunk 2 above)
Comment (thedavidmeister) [reply crates/hdk/src/link.rs:38]: @jost-s live is a DHT concept, is this the best place to explain it?

Comment (jost-s) [reply crates/hdk/src/link.rs:38]: Is this a rhethorical question? =D

Comment (thedavidmeister) [reply crates/hdk/src/link.rs:38]: @jost-s it doesn't seem like a good place to document it to me, i don't see ppl going to the link docs to understand how the DHT works

Comment (jost-s) [reply crates/hdk/src/link.rs:38]: Sure, agreed

**Pull request merged**